### ISSUE TITLE
Make a decision on FML inclusion in L3 pacakges

### ISSUE THREAD

**Issue by jingtang10** [WorldHealthOrganization/smart-ig-starter-kit#58]
FHIR mapping langauge is currently used to author strucuture maps. But we need to make a separate decision on if/how it should be included in L3 packages.

This has an impact on how L4 uses L3.

**Comment by costateixeira** [comment]
good point. I'd lean to make structuremaps mandatory and FML inside the structuremaps (which is what the FHIR publisher now does). 
Reason for this is that having a compilable, error-free structuremap is (IMO) a responsibility of the L3 author, and part of the criteria, not something that is for the L4 implementer to figure out.

**Comment by litlfred** [comment]
> FHIR mapping langauge is currently used to author strucuture maps. But we need to make a separate decision on if/how it should be included in L3 packages.
> 
> This has an impact on how L4 uses L3.

I think we need to also be clear as to what are the packages.   The "WHO Normative Release of L3 Artifacts" package should (as @costateixeira suggests) have everything in there, Logical Models and Structure Maps are a part of the normative content.  Note the FML source should be included in the StructureMap resources when it is there.

 It is up to an implementation community (e.g. Android FHIR SDK) to determine if it wants all of that or not in an "L4 Implementation Package" or if it wants to, for example, optimize away the use of FMLs/SMs to and from LM or if wants to develop their own ones.

**Comment by brynrhodes** [comment]
Can we add this to the next authoring call agenda? I have a different understanding, I had thought that the published implementation guide content was focused on L3, not L2 content?
